objective-c/HelloWorld/HelloWorld/Base.lproj/LaunchScreen.xib

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
You can get rid of this file by searching in your XCode project (the magnifying glass of the left panel) for "LaunchScreen". It'll find it mentioned in a .plist I think. You can replace it with "Main" (your storyboard) and then safely delete this file.